README.md

```diff
@@ -471,6 +471,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
 
 ## Code Generation
 
+  * [Appinvento](https://appinvento.io/) — Build your application’s backend easily with AppInvento. you can have full control and ownership of the source code. Enjoy the added benefit of automatically generated APIs for seamless integration.
```

Comment:
Please mention what is free.

Reply:
@ripienaar , mention free features in detail. please check once

Reply:
You've rewritten the entire entry now again into full marketing mode, this is not an advert I suggest:

```
Build your application’s backend easily with AppInvento. you can have full control and ownership of the source code. Enjoy the added benefit of automatically generated APIs for seamless integration. Free plan includes three projects, five tables, and a Google add-on.
```

Reply:
@ripienaar , can we go with this?

AppInvento is a free No code app builder. In the automatically generated backend code, users have complete access to the source code along with unlimited APIs and routes, allowing for extensive integration. The free plan includes three projects, five tables, and a Google add-on.

Reply:
Sure thanks.

Reply:
@ripienaar , Please review the PR, i have done changes.